### ISSUE TITLE
Newest npm releases not built (missing dist folder)

### ISSUE THREAD

**Issue by dtabuenc** [testing-library/user-event#494]
It seems to me that releases 12.2.1 and 12.2.2 do not include built version of the code in the dist/ folder. Is there  a problem with the CI build?

**Comment by nickmccurdy** [comment]
12.2.2 should fix the package distribution. Can you please post a reproduction of this failing with that version?

**Comment by kentcdodds** [comment]
Yep, the latest version fixes this: https://unpkg.com/browse/@testing-library/user-event@12.2.2/
